components/taupage.rst

```diff
@@ -688,3 +688,18 @@ You can build your own Taupage AMI using the code from the repository on GitHub
 In the repository you will find a configuration (config-stups-example.sh) file which you'll have to adjust to your needs.
 
 See :ref:`taupage-ami-creation` for details.
+
+Support for NVIDIA GPUs
++++++++++++++++++++++++
+
+Taupage supports the use of NVIDIA CUDA-enabled GPUs if these are available on the EC2 host (e.g. G2 or P2 instances). In this case, `nvidia-docker` (https://github.com/NVIDIA/nvidia-docker) is used as a drop-in replacement for Docker. This creates a Docker volume which contains the CUDA driver files installed on the host. This volume, as well as the required NVIDIA device nodes, are mounted into the running container allowing GPU-enabled applications to be run.
```

Comment:
hyperlink the text instead of having the URL in the text: http://www.sphinx-doc.org/en/stable/rest.html#hyperlinks